qa/vagrant/src/test/resources/packaging/tests/75_bad_data_paths.bats

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bats
+
+# Tests data.path settings which in the past have misbehaving, leaking the
+# default.data.path setting into the data.path even when it doesn't belong.
+
+# WARNING: This testing file must be executed as root and can
+# dramatically change your system. It removes the 'elasticsearch'
+# user/group and also many directories. Do not execute this file
+# unless you know exactly what you are doing.
```

Comment:
But I'll leave it on my list and take it when I can if no one else gets to it first.